.pre-commit-config.yaml

```diff
@@ -20,7 +20,15 @@ repos:
     rev: 7.0.0
     hooks:
       - id: flake8
-  - repo: https://github.com/pre-commit/mirrors-eslint
-    rev: v9.2.0
+  - repo: local
     hooks:
-      - id: eslint
+    - id: eslint
+      name: eslint
+      language: node
+      additional_dependencies:
+      - 'eslint@9.2.0'
+      - '@eslint/js@9.2.0'
+      - 'globals@15.2.0'
```

Comment:
Is it not an issue in `pre-commit` since dependencies are defined in `package.json`? :thinking:

Reply:
It seems that it's documented that you need to explicitly define dependencies  in the `pre-commit` configuration.

Reply:
pre-commit uses isolated environments, it never installs from your dependency files like `package.json`.